packages/rpcs/wallet_enrichConfluxTx/index.js

```diff
@@ -41,9 +41,13 @@ export const main = async ({
 
   if (to) {
     decoded = decode(to)
-    if (decoded.type !== 'contract' || !data)
+    if (
+      decoded.type === 'user' ||
+      decoded.hexAddress === '0x0000000000000000000000000000000000000000' ||
```

Comment:
this means that if the to address is Zero address, we will treat it as a common tx?

Reply:
yes